src/_data/examples.yml

```diff
@@ -35,11 +35,9 @@ showcase:
 
 - section: showcase
   slug: videosphere
-  scene_url: https://aframe.io/aframe/examples/boilerplate/360-video/
-  source_url: https://github.com/aframevr/aframe/blob/v0.7.0/examples/boilerplate/360-video/index.html
+  scene_url: https://aframe-360-video-example.glitch.me/
+  source_url: https://glitch.com/~aframe-360-video-example
```

Comment:
https://glitch.com/edit/#!/aframe-360-video-example is the actual URL for View Source

Reply:
probably makes most sense to also link directly to the `index.html` file: https://glitch.com/edit/#!/aframe-360-video-example?path=index.html:1:0

Reply:
I was following the example of the other two glitches I saw in there for consistency ... and there is some useful information in the README that gets displayed.